doc/multiple_ports.rst

```diff
@@ -59,55 +59,89 @@ Enable and start network daemon::
     sudo systemctl enable systemd-networkd
     sudo systemctl start systemd-networkd
 
-Run ``ifconfig`` to check that the bridge is up and its network interfaces are all up::
-
-    pi@pndevice-pi:~$ ifconfig
-    br0: flags=4419<UP,BROADCAST,RUNNING,PROMISC,MULTICAST>  mtu 1500
-        ether c2:38:f3:a6:0a:66  txqueuelen 1000  (Ethernet)
-        RX packets 6871  bytes 614728 (600.3 KiB)
-        RX errors 0  dropped 12  overruns 0  frame 0
-        TX packets 929  bytes 288939 (282.1 KiB)
-        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
-
-    eth0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
-        ether b8:27:eb:67:14:8a  txqueuelen 1000  (Ethernet)
-        RX packets 0  bytes 0 (0.0 B)
-        RX errors 0  dropped 0  overruns 0  frame 0
-        TX packets 0  bytes 0 (0.0 B)
-        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
-
-    eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
-        inet6 fe80::c641:1eff:fe82:f67f  prefixlen 64  scopeid 0x20<link>
-        ether c4:41:1e:82:f6:7f  txqueuelen 1000  (Ethernet)
-        RX packets 6901  bytes 617674 (603.1 KiB)
-        RX errors 0  dropped 0  overruns 0  frame 0
-        TX packets 972  bytes 293401 (286.5 KiB)
-        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
-
-    lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
-        inet 127.0.0.1  netmask 255.0.0.0
-        inet6 ::1  prefixlen 128  scopeid 0x10<host>
-        loop  txqueuelen 1000  (Local Loopback)
-        RX packets 0  bytes 0 (0.0 B)
-        RX errors 0  dropped 0  overruns 0  frame 0
-        TX packets 0  bytes 0 (0.0 B)
-        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
-
-Configuration of p-net stack and sample application 
+The sample app will set the ``br0`` bridge state to ``UP`` when started.
```

Comment:
Is this true? Is the application controlling the network interfaces or just using them

Reply:
That is what happened when I tried.

Reply:
Modified the text. Added a separate file to bring the bridge up. Added paragraph on packet forwarding via bridge.
Done.